<ui version="4.0" >
 <class>MainGui</class>
 <widget class="QMainWindow" name="MainGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qtpfsgui</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons.qrc" >:/new/prefix1/images/qtpfsgui.png</iconset>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="OptionsAction" />
    <addaction name="separator" />
    <addaction name="Transplant_Exif_Data_action" />
    <addaction name="actionBatch_Tone_Mapping" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="documentationAction" />
    <addaction name="actionWhat_s_This" />
    <addaction name="separator" />
    <addaction name="actionAbout_Qtpfsgui" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menuWindows" >
    <property name="title" >
     <string>&amp;Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuHDR_Histogram" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Hdr &amp;Histogram</string>
     </property>
     <addaction name="Fit_to_dynamic_range" />
     <addaction name="Low_dynamic_range" />
     <addaction name="Shrink_dynamic_range" />
     <addaction name="Extend_dynamic_range" />
     <addaction name="Decrease_exposure" />
     <addaction name="Increase_exposure" />
    </widget>
    <widget class="QMenu" name="menuToolbars" >
     <property name="title" >
      <string>&amp;Toolbars</string>
     </property>
     <addaction name="actionText_Under_Icons" />
     <addaction name="actionIcons_Only" />
     <addaction name="actionText_Alongside_Icons" />
     <addaction name="actionText_Only" />
     <addaction name="separator" />
    </widget>
    <addaction name="fitToWindowAct" />
    <addaction name="normalSizeAct" />
    <addaction name="zoomInAct" />
    <addaction name="zoomOutAct" />
    <addaction name="menuHDR_Histogram" />
    <addaction name="separator" />
    <addaction name="menuToolbars" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileNewAction" />
    <addaction name="fileOpenAction" />
    <addaction name="fileSaveAsAction" />
    <addaction name="actionSave_Hdr_Preview" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="menuImage" >
    <property name="title" >
     <string>&amp;Image</string>
    </property>
    <addaction name="rotateccw" />
    <addaction name="rotatecw" />
    <addaction name="separator" />
    <addaction name="action_Projective_Transformation" />
    <addaction name="actionResizeHDR" />
    <addaction name="separator" />
    <addaction name="TonemapAction" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuImage" />
   <addaction name="menuView" />
   <addaction name="menuSettings" />
   <addaction name="menuWindows" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Common operations</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="fileNewAction" />
   <addaction name="fileOpenAction" />
   <addaction name="fileSaveAsAction" />
   <addaction name="separator" />
   <addaction name="TonemapAction" />
   <addaction name="separator" />
   <addaction name="fileExitAction" />
  </widget>
  <action name="fileNewAction" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/newfile.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New Hdr...</string>
   </property>
   <property name="iconText" >
    <string>New Hdr...</string>
   </property>
   <property name="toolTip" >
    <string>Create an Hdr from a set of JPEGs or RAWs, or 8 or 16 bit TIFF files</string>
   </property>
   <property name="statusTip" >
    <string>Create an Hdr from a set of JPEGs or RAWs, or 8 or 16 bit TIFF files.</string>
   </property>
   <property name="whatsThis" >
    <string>Create an Hdr from a set of JPEGs or RAWs, or 8 or 16 bit TIFF files.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fileOpenAction" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open Hdr...</string>
   </property>
   <property name="iconText" >
    <string>Open Hdr....</string>
   </property>
   <property name="toolTip" >
    <string>Load an existing Hdr image file (OpenEXR, Radiance RGBE, PFS stream, RAW or TIFF)</string>
   </property>
   <property name="statusTip" >
    <string>Load an existing Hdr image file (OpenEXR, Radiance RGBE, PFS stream, RAW or TIFF).</string>
   </property>
   <property name="whatsThis" >
    <string>Load an existing Hdr image file (OpenEXR, Radiance RGBE, PFS stream, RAW or TIFF).</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileSaveAsAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save Hdr as...</string>
   </property>
   <property name="iconText" >
    <string>Save Hdr as...</string>
   </property>
   <property name="toolTip" >
    <string>Saves the Hdr (OpenEXR, Radiance RGBE, PFS stream, or 32bit or LogLuv TIFF)</string>
   </property>
   <property name="statusTip" >
    <string>Saves the Hdr (OpenEXR, Radiance RGBE, PFS stream, or 32bit or LogLuv TIFF)</string>
   </property>
   <property name="whatsThis" >
    <string>Saves the Hdr (OpenEXR, Radiance RGBE, PFS stream, or 32bit or LogLuv TIFF)</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="TonemapAction" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/tonemap.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Tonemap the Hdr...</string>
   </property>
   <property name="iconText" >
    <string>Tonemap the Hdr</string>
   </property>
   <property name="toolTip" >
    <string>Tonemap the Hdr into a common LDR (i.e. JPEG or PNG) format</string>
   </property>
   <property name="statusTip" >
    <string>Tonemap the Hdr into a common LDR (i.e. JPEG or PNG) format</string>
   </property>
   <property name="whatsThis" >
    <string>Tonemap the Hdr into a common LDR (i.e. JPEG or PNG) format</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip" >
    <string>Exit</string>
   </property>
   <property name="statusTip" >
    <string>Exit</string>
   </property>
   <property name="whatsThis" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="rotateccw" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/rotate_left.png</iconset>
   </property>
   <property name="text" >
    <string>Rotate C&amp;ounterClockWise</string>
   </property>
   <property name="iconText" >
    <string>Rotate CounterClockWise</string>
   </property>
   <property name="toolTip" >
    <string>Rotate Counterclockwise the Hdr</string>
   </property>
   <property name="statusTip" >
    <string>Rotate Counterclockwise the Hdr</string>
   </property>
   <property name="whatsThis" >
    <string>Rotate Counterclockwise the Hdr</string>
   </property>
   <property name="shortcut" >
    <string>&lt;</string>
   </property>
  </action>
  <action name="rotatecw" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/rotate_right.png</iconset>
   </property>
   <property name="text" >
    <string>Rotate C&amp;lockWise</string>
   </property>
   <property name="iconText" >
    <string>Rotate ClockWise</string>
   </property>
   <property name="toolTip" >
    <string>Rotate ClockWise the Hdr</string>
   </property>
   <property name="statusTip" >
    <string>Rotate ClockWise the Hdr</string>
   </property>
   <property name="whatsThis" >
    <string>Rotate ClockWise the Hdr</string>
   </property>
   <property name="shortcut" >
    <string>></string>
   </property>
  </action>
  <action name="Low_dynamic_range" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Low Dynamic Range</string>
   </property>
   <property name="shortcut" >
    <string>L</string>
   </property>
  </action>
  <action name="Fit_to_dynamic_range" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Fit to Dynamic Range</string>
   </property>
   <property name="shortcut" >
    <string>\</string>
   </property>
  </action>
  <action name="Shrink_dynamic_range" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Shrink Dynamic Range</string>
   </property>
   <property name="shortcut" >
    <string>[</string>
   </property>
  </action>
  <action name="Extend_dynamic_range" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Extend Dynamic Range</string>
   </property>
   <property name="shortcut" >
    <string>]</string>
   </property>
  </action>
  <action name="Decrease_exposure" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Decrease Exposure</string>
   </property>
   <property name="shortcut" >
    <string>0</string>
   </property>
  </action>
  <action name="Increase_exposure" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Increase Exposure</string>
   </property>
   <property name="shortcut" >
    <string>9</string>
   </property>
  </action>
  <action name="documentationAction" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/help.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Documentation...</string>
   </property>
   <property name="iconText" >
    <string>About...</string>
   </property>
   <property name="toolTip" >
    <string>Show Qtpfsgui Help</string>
   </property>
   <property name="statusTip" >
    <string>Show Qtpfsgui Help</string>
   </property>
   <property name="whatsThis" >
    <string>Show Qtpfsgui Help</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="normalSizeAct" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/viewmag1.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Normal Size</string>
   </property>
   <property name="toolTip" >
    <string>View Hdr at Normal Size</string>
   </property>
   <property name="statusTip" >
    <string>View Hdr at Normal Size</string>
   </property>
   <property name="whatsThis" >
    <string>View Hdr at Normal Size</string>
   </property>
   <property name="shortcut" >
    <string>O</string>
   </property>
  </action>
  <action name="zoomInAct" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/viewmag_inc.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom &amp;In</string>
   </property>
   <property name="statusTip" >
    <string>Zoom In</string>
   </property>
   <property name="whatsThis" >
    <string>Zoom In</string>
   </property>
   <property name="shortcut" >
    <string>+</string>
   </property>
  </action>
  <action name="zoomOutAct" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/viewmag_dec.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom &amp;Out</string>
   </property>
   <property name="statusTip" >
    <string>Zoom Out</string>
   </property>
   <property name="whatsThis" >
    <string>Zoom Out</string>
   </property>
   <property name="shortcut" >
    <string>-</string>
   </property>
  </action>
  <action name="fitToWindowAct" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/viewmagfit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Fit to Window</string>
   </property>
   <property name="statusTip" >
    <string>Shrink the Hdr to fit its containing window</string>
   </property>
   <property name="whatsThis" >
    <string>Shrink the Hdr to fit its containing window</string>
   </property>
   <property name="shortcut" >
    <string>W</string>
   </property>
  </action>
  <action name="OptionsAction" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/options.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Preferences...</string>
   </property>
   <property name="iconText" >
    <string>Qtpfsgui Options</string>
   </property>
   <property name="toolTip" >
    <string>Set the various options of Qtpfsgui</string>
   </property>
   <property name="statusTip" >
    <string>Set the various options of Qtpfsgui.</string>
   </property>
   <property name="whatsThis" >
    <string>Set the various options of Qtpfsgui.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionResizeHDR" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/scale.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Resize the Hdr...</string>
   </property>
   <property name="iconText" >
    <string>Resize the Hdr</string>
   </property>
   <property name="toolTip" >
    <string>Resize (shrink or expand) the Hdr</string>
   </property>
   <property name="statusTip" >
    <string>Resize (shrink or expand) the Hdr</string>
   </property>
   <property name="whatsThis" >
    <string>Resize (shrink or expand) the Hdr</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="Transplant_Exif_Data_action" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/copyexif.png</iconset>
   </property>
   <property name="text" >
    <string>Copy &amp;Exif Data...</string>
   </property>
   <property name="iconText" >
    <string>Copy Exif Data</string>
   </property>
   <property name="toolTip" >
    <string>Copy Exif Data between two set of files</string>
   </property>
   <property name="statusTip" >
    <string>Copy Exif Data between two set of files</string>
   </property>
   <property name="whatsThis" >
    <string>Copy Exif Data between two set of files</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAlign_Images" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Align Images...</string>
   </property>
   <property name="toolTip" >
    <string>WORK IN PROGRESS</string>
   </property>
   <property name="statusTip" >
    <string>WORK IN PROGRESS</string>
   </property>
   <property name="whatsThis" >
    <string>WORK IN PROGRESS</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="actionTile" >
   <property name="text" >
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="actionCascade" >
   <property name="text" >
    <string>&amp;Cascade</string>
   </property>
   <property name="iconText" >
    <string>Cascade</string>
   </property>
   <property name="toolTip" >
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionToolbars" >
   <property name="text" >
    <string>Toolbars</string>
   </property>
  </action>
  <action name="actionText_Under_Icons" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Text &amp;Under Icons</string>
   </property>
  </action>
  <action name="actionIcons_Only" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Icons Only</string>
   </property>
  </action>
  <action name="actionText_Only" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Text Only</string>
   </property>
  </action>
  <action name="actionText_Alongside_Icons" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Text &amp;Alongside Icons</string>
   </property>
  </action>
  <action name="actionBatch_Tone_Mapping" >
   <property name="text" >
    <string>&amp;Batch Tone Mapping...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionWhat_s_This" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/new/prefix1/images/whatsthis.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;What's This?</string>
   </property>
   <property name="toolTip" >
    <string>Get information about the graphical elements</string>
   </property>
   <property name="shortcut" >
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="actionAbout_Qtpfsgui" >
   <property name="text" >
    <string>About &amp;Qtpfsgui</string>
   </property>
  </action>
  <action name="actionSave_Hdr_Preview" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Save Hdr &amp;Preview...</string>
   </property>
   <property name="toolTip" >
    <string>Save the current Hdr Preview to a Ldr file.</string>
   </property>
   <property name="statusTip" >
    <string>Save the current Hdr Preview to a Ldr file.</string>
   </property>
   <property name="whatsThis" >
    <string>Save the current Hdr Preview to a Ldr file.</string>
   </property>
  </action>
  <action name="action_Projective_Transformation" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Projective Transformation...</string>
   </property>
   <property name="toolTip" >
    <string>Apply a projective transformation to the current Hdr.</string>
   </property>
   <property name="statusTip" >
    <string>Apply a projective transformation to the current Hdr.</string>
   </property>
   <property name="whatsThis" >
    <string>Apply a projective transformation to the current Hdr.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections/>
</ui>
